app.js

```diff
@@ -404,60 +404,76 @@ let DNSResult = create({
 
   render: function() {
     let j = this.props.json;
-    if (j.SRVRecords == null) {
-      return (
-        <div className="dns">
-          <h2>No SRV Records</h2>
-        </div>
-      );
+
+    let dnsResultTitle = <> <h3>DNS records for {j.SRVCName}</h3> </>
```

Comment:
note that the `<> ... </>` is redundant here:

```suggestion
    let dnsResultTitle = <h3>DNS records for {j.SRVCName}</h3>;
```